app/assets/stylesheets/asset-nav-bar.scss

```diff
@@ -0,0 +1,54 @@
+.asset-nav-bar li a {
```

Comment:
Can be nesting the li components inside
 .asset-nav-bar {
     li {
That's the beauty of sass